### DISCUSSION TITLE
Release

### DISCUSSION THREAD

**Discussion by adouglass** [flutter-stripe/flutter_stripe#159] [General]
I'd like to incorporate the fix to #14 into my app-- and was wondering whether you'd be dropping a new release that incorporates that change? Is there a specific release schedule for this framework?

**Comment by remonh87** [comment]
We are at this moment syncing with the latest Stripe sdk and when this is finished we will launch a new version. We expect to release a new version around next week. For the time being you can use the `main` branch as dependency. Also notice that this issue also needs a new Flutter version.
